### PR TITLE
an introductory post about erc-20

### PR DESCRIPTION
**Pull request by ajingene**
This is more or less it. I don't expect any changes, but I'd like to review it again tomorrow after sleeping on it. Also to apply any comments you might have.

### CODE REVIEW

File: _posts/2021-05-16-erc20-token.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+for tokens" even mean?  Why is such a standard even necessary or
+useful?
+</p>
+
+<p>
+Let's start with some nomenclature.  "ERC" stands for "Ethereum
+Request for Comments" and "20" is a number denoting this particular
+ERC, about a certain kind of tokens in Ethereum.  ERC is a term
+reminiscent of RFCs ("Request for Comments"), a similar concept in the
+context of Internet standards.  Also, note that ERC-20 is a standard
```
Comment (josipkoncurat) [inline _posts/2021-05-16-erc20-token.html:25]: Although a nice comparison, for some not knowing anything about internet standards it's not really useful. I would maybe put a sentence or two on what does "Request for comments" means in this context. If it has any meaning at all.

File: _posts/2021-05-16-erc20-token.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+</p>
+
+<h3 id="org857ab55">What are tokens and how are they different from coins?</h3>
+<p>
+Coins have their own blockchain, tokens exist on a different
+blockchain that has its own native coin.  In Ethereum, the native coin
+is called 'ether' and tokens are created through smart contracts on
+the Ethereum blockchain.  (Note that tokens were created even before
+the standard ERC-20 was made.  Or rather, the fact that tokens were
+being created was the incentive to create the standard)
```
Comment (josipkoncurat) [inline _posts/2021-05-16-erc20-token.html:38]: Missing a dot(.)

File: _posts/2021-05-16-erc20-token.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+you're finished and people start using your wallet application,
+suppose you decide to offer that same functionality for a different
+token.  It would be great if you didn't have to rewrite parts of your
+application just for that other token, and then again for a third
+token.  If all those tokens follow a standard, then you don't have to.
+You can use the application for all
+standard-compliant tokens, including those that weren't created at the moment
+you coded your wallet.  Why?  Because they behave in a
+standardized fashion, and you code your wallet with expectations of
+such behavior.
```
Comment (josipkoncurat) [inline _posts/2021-05-16-erc20-token.html:112]: I would use a different word than "code"

**Review by josipkoncurat — COMMENTED** [review]
A nice intro into standards, for non-programmers the notion of interfaces I am sure is very confusing.

**Comment by ajingene** [comment]
ok, done

**Pull request merged**